source/rmarkdown/analyses_diversity/_child_model_observed_richness.Rmd

```diff
@@ -75,3 +135,207 @@ marginaleffects::plot_predictions(
 ```
 
 
+#### Met physicochemische data
+
+Stepwise model selection:
+
+```{r}
+variables_no_collinearity <- c(
+  "total_count", 
+  "landgebruik", 
+  "diepte", 
+  "cmon_plot_id", 
+  "c_density", 
+  "cn_stockbased", 
+  "swc_vol", 
+  "ph_kcl", 
+  "bd"
+)
+
+nr1 <- nrow(data_subset)
+cat(
+  paste(
+    "De originele dataset heeft",
+    nr1,
+  "rijen.\n"
+  )
+)
+data_subset <- data_subset[
+  complete.cases(data_subset[ , variables_no_collinearity]), ]
+nr2 <- nrow(data_subset)
+
+cat(
+  paste(
+    "Na verwijderen van rijen waarvan er ontbrekende gegevens zijn voor
+    één van de covariaten, heeft de dataset",
+    nr2,
+  "rijen.\n"
+  )
+)
+```
+
+
+We voegen alle niet collineaire fysicochemische variabelen toe aan het basismodel.
+Voor pH gebruiken we een kwadratische effect.
+Dit wordt het `full model`.
+
+```{r extend-model-additional-predictors-{{group}}-{{primerset}}-{{unit}}}
+form_richness_full <- update(
+  form_richness,
+  . ~ . 
+  + swc_vol
+  + c_density
+  + cn_stockbased
+  + swc_vol
```

Comment:
@hansvancalster waarom zit `swc_vol` hier 2 keer in het extended / full model?

Reply:
wel eentje dat geen gevolgen zal hebben vermoedelijk

Reply:
> wel eentje dat geen gevolgen zal hebben vermoedelijk

Dit lijkt te kloppen, ik zie in de output na https://github.com/inbo/mbag-bodem/pull/41 voorlopig geen verschil